static/third/bootstrap/css/bootstrap.css

```diff
@@ -960,6 +960,10 @@ table {
   vertical-align: top;
   border-top: 1px solid #dddddd;
 }
+.table td:not(:first-child) {
+  padding-right: 76px;
+  text-align: center;
+}
```

Comment:
These changes need to be in a non-third-party file.  Also, the `padding-right` is a clear sign that we don't have the CSS logically correct.